packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,351 @@ Specific property types support a subset of [JSON Schema validations](https://to
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                },
+                "lastName": {
+                    "title": "Last name",
+                    "type": "string",
+                    "required": true,
+                    "minLength": 1,
+                    "maxLength": 50,
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                },
+                "email": {
+                    "title": "Primary email",
+                    "type": "string",
+                    "required": true,
+                    "format": "email",
+                    "permissions": [
+                        {
+                            "principal": "SELF",
+                            "action": "READ_WRITE"
+                        }
+                    ]
+                }
+            },
+            "required": [
+                "login",
+                "firstName",
+                "lastName",
+                "email"
+            ]
+        },
+        "custom": {
+            "id": "#custom",
+            "type": "object",
+            "properties": {
+              "twitterUserName": {
+                  "title": "Twitter username",
+                  "description": "User's username for twitter.com",
+                  "type": "string",
+                  "required": false,
+                  "unique": "UNIQUE_VALIDATED",
+                  "minLength": 1,
+                  "maxLength": 20,
+                  "permissions": [
+                      {
+                          "principal": "SELF",
+                          "action": "READ_WRITE"
+                      }
+                  ]
+              }
+            },
+            "required": []
+        }
+    },
+    "type": "object",
+    "properties": {
+        "profile": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/base"
+                },
+                {
+                    "$ref": "#/definitions/custom"
+                }
+            ]
+        }
+    }
+}
+```
+
+### Update existing custom property to be unique
+
+You can use the [update user profile schema property](#update-user-profile-schema-property) request or the [update group profile schema property](#update-group-profile-schema-property) request to mark existing custom user or group properties as unique, by specifying `"unique": true` on the properties that need to be marked as unique.
+
+Once the request to mark existing custom properties as unique is submitted, an asynchronous validation check is performed to make sure that there are no existing duplicate entries. If you have significant user or group records, the validation can take some time.
```

Comment:
```suggestion
After the request to mark existing custom properties as unique is submitted, an asynchronous validation check is performed to make sure that there are no existing duplicate entries. If you have significant user or group records, the validation can take some time.
```

Reply:
"significant user or group records" --> "a significant number of users or groups"
(Let's not get into what would make an individual user or group record be significant.) :trollface: